agents/postgres/pgstatmonitor/stat_monitor_cache.go

```diff
@@ -0,0 +1,237 @@
+// pmm-agent
+// Copyright 2019 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pgstatmonitor
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/percona/pmm-agent/utils/truncate"
+
+	"github.com/AlekSi/pointer"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"gopkg.in/reform.v1"
+
+	"github.com/percona/go-mysql/query"
```

Comment:
Shouldn't it be https://github.com/lfittl/pg_query_go here? 
https://github.com/percona/pmm-agent/pull/183#discussion_r488456480

Reply:
Yes, right I clone it from 6460 and there it wasnt still fixed. Thanks.